src/pages/MovieDetailsPage.js

```diff
@@ -55,6 +56,10 @@ export default function SeriesDetailsPage() {
     </Wrapper>
   );
 
+  function goBack() {
```

Comment:
When i am right you could also just write onClick={() => navigate(-1)} and then you dont need the function 😊

Reply:
Thanks a lot! It works to :)